src/libraries/System.Security.Cryptography.Pkcs/tests/SignedCms/SignerInfoTests.cs

```diff
@@ -819,7 +820,7 @@ public static void AddCounterSigner_ECDSA(SubjectIdentifierType identifierType,
             // DSA PKIX signature format is a DER SEQUENCE.
             Assert.Equal(0x30, signature[0]);
 
-            // ECDSA Oids are all under 1.2.840.10045.4.
+            // CDSA Oids are all under 1.2.840.10045.4.
```

Comment:
```suggestion
            // ECDSA Oids are all under 1.2.840.10045.4.
```

That E that you deleted was "important" :)